src/services/item-geolocation/types.ts

```diff
@@ -0,0 +1,12 @@
+import { DiscriminatedItem } from '..';
+import { UUID } from '@/types';
+
+export type ItemGeolocation = {
+  id: UUID;
+  lat: number;
+  lng: number;
+  item: DiscriminatedItem;
+  country: string | null;
```

Comment:
If it is impossible to insert an ItemGeolocation without the country, the country could be a string only ?  If not, just ignore my comment.

I'm talking about this comment: https://github.com/graasp/graasp/pull/777#discussion_r1465129257

Reply:
Yeah I changed since it's possible to add a point in the sea (without a country) I guess 🤷‍♀️